metadata.json

```diff
@@ -41,6 +49,9 @@
     }
   ],
   "tags": [
-    "fail2ban"
+    "fail2ban",
+    "debian",
+    "ubuntu",
+    "redhat"
```

Comment:
Please remove lines 53 to 55, because the operating system might be bad for the sore at Puppet Forge.